code/modules/mob/living/carbon/human/species/necromorph/infector/conversion.dm

```diff
@@ -130,6 +130,10 @@
 	var/list/options = list(SPECIES_NECROMORPH_SLASHER = (9.5 / compatibility),
 	SPECIES_NECROMORPH_SLASHER_ENHANCED = (1 * compatibility))
 
+	//Monkey?
+	if (biomass == 7)
```

Comment:
and if the size category is whatever monkeys are, just add this option and return without executing the rest of the proc

Reply:
Can't check for size, it's considered a humanoid. The second proc overwrites the first from testing.

Reply:
I can however, check directly for whether or not it's a monkey with a selector? But I'd  have to test. biomass is the easiest.

Reply:
Monkeys are size MOB_SMALL

humans are MOB_MEDIUM
https://live.staticflickr.com/65535/51533256355_40bfba0a5d_o.png

check the mob_size var on the mob

Reply:
I've tried before, monkeys literally turned into leapers, pukers, etc before. It 100% treats them according to the second proc, not the size proc.

Reply:
I physically can't go by mob size without changing the second proc as it takes precedent over the first because it detects a human, the parent of the monkey mob.